core/lexicon/nl/login.inc.php

```diff
@@ -1,8 +1,8 @@
 <?php
```

Comment:
Translations are normally handled via CrowdIn. There's a chance this file will get overwritten after it is synchronised, so they may need to be re-applied...